extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -1097,7 +1097,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       $(el).addClass('no_pgp');
       if (info?.info.name) {
         recipient.name = info.info.name;
-        $(el).find('.recipient-name').text(Xss.escape(info.info.name));
+        $(el).find('.recipient-name').text(Xss.stripEmojis(info.info.name));
```

Comment:
no need for `stripEmojis` here, let's change it back to `escape`

Reply:
ok, will do - thanks.